Unosquare.FFME/Engine/BlockRenderingWorker.cs

```diff
@@ -17,32 +16,21 @@ namespace Unosquare.FFME.Engine
     /// Implements the block rendering worker.
     /// </summary>
     /// <seealso cref="IMediaWorker" />
-    internal sealed class BlockRenderingWorker : IntervalWorkerBase, IMediaWorker, ILoggingSource
+    internal sealed class BlockRenderingWorker : WorkerBase, IMediaWorker, ILoggingSource
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Implement 'IDisposable' in this class and use the 'Dispose' method to call 'Dispose' on 'QuantumWaiter'.](https://app.codacy.com/app/UnosquareLabs/ffmediaelement/pullRequest?prid=4726833)